Source/JavaScriptCore/bytecode/ProxyObjectAccessCase.cpp

```diff
@@ -109,27 +109,29 @@ void ProxyObjectAccessCase::emit(AccessGenerationState& state, MacroAssembler::J
 
     jit.store32(CCallHelpers::TrustedImm32(numberOfParameters), calleeFrame.withOffset(CallFrameSlot::argumentCountIncludingThis * sizeof(Register) + PayloadOffset));
 
+    jit.storeTrustedValue(jsUndefined(), calleeFrame.withOffset(virtualRegisterForArgumentIncludingThis(0).offset() * sizeof(Register)));
```

Comment:
Just showing my ignorance here. We are always storing an undefined here, so how is target ever anything other than undefined?

Reply:
This is new code :)
`target` becomes the first argument now (this argument 0 is `this` in JSC, (that's why the function name is `virtualRegisterForArgumentIncludingThis`)).

Reply:
I see now, and we don’t need a `this` so we can just pass `undefined`.